### PR TITLE
Fix MeterRegistry cast exception caused by the open telemetry java agent

### PR DESCRIPTION
**Pull request by EdwardKuenen**
https://github.com/rh-messaging/artemis-prometheus-metrics-plugin/issues/12

### CODE REVIEW

File: artemis-prometheus-metrics-plugin/src/main/java/com/redhat/amq/broker/core/server/metrics/plugins/ArtemisPrometheusMetricsPlugin.java [hunk 1]
```diff
@@ -29,8 +29,8 @@ public class ArtemisPrometheusMetricsPlugin implements ActiveMQMetricsPlugin {
    MeterRegistry meterRegistry;
 
    @Override
-   public ActiveMQMetricsPlugin init(Map<String, String> options) {
-      this.meterRegistry = new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
+   public ActiveMQMetricsPlugin init(final Map<String, String> options) {
```
Comment (brusdev) [inline artemis-prometheus-metrics-plugin/src/main/java/com/redhat/amq/broker/core/server/metrics/plugins/ArtemisPrometheusMetricsPlugin.java:32]: This fix LGTM, could you only revert this change if not required?

**Comment by brusdev** [comment]
Do you mind if I amend your commit message to `Fix MeterRegistry cast exception caused by the open telemetry java agent`?

**Comment by EdwardKuenen** [comment]
I hope I did it correct. It is my first PR on an open source project ;)

**Comment by brusdev** [comment]
@EdwardKuenen I squashed your commit using `git rebase -i HEAD~4` (of course I didn't change the commit author) and then I pushed your branch with the force option using `git push ekuenen only-otel-agent-fix -f`, I'll merge this PR in 1h unless you have any concerns.

**Comment by jbertram** [comment]
LGTM
